lib/galaxy/datatypes/sequence.py

```diff
@@ -844,19 +847,45 @@ def quality_check(lines):
 
 
 class FastqSolexa(Fastq):
-    """Class representing a FASTQ sequence ( the Solexa variant )"""
+    """Class representing a FASTQ sequence ( the Solexa variant )
+
+    solexa scored quality values -5:40 represented by ASCII 59:104
+    """
     edam_format = "format_1933"
     file_ext = "fastqsolexa"
 
+    @staticmethod
+    def quality_check(lines):
+        """Presuming lines are lines from a fastq file, return True if the qualities are compatible with sanger encoding"""
+        for line in islice(lines, 3, None, 4):
+            if not all(_ >= ';' and _ <= 'h' for _ in line[0]):
```

Comment:
I think, the underscore, by convention, is designated for unused variables (placeholders in unpacking, listcomps, etc.). (PEP 640 is proposing to formalize this). But here the `_` variable is used for comparison and, therefore, has meaning. My suggestion would be to give it a real/descriptive name (a single-character name would be better, I think).

Reply:
Sure. I think I just copy pasted this from `FastqSanger`. Fixed it also there.

Reply:
Thank you! Much more readable now, I think.